util/src/main/java/tech/pegasys/artemis/util/hashtree/HashTreeUtil.java

```diff
@@ -86,15 +86,17 @@ public static Bytes32 hash_tree_root(SSZTypes sszType, List bytes) {
           return hash_tree_root_list_composite_type((List<Merkleizable>) bytes, bytes.size());
         }
         break;
+      case TUPLE_OF_COMPOSITE:
+        if (!bytes.isEmpty() && bytes.get(0) instanceof Bytes32) {
+          return hash_tree_root_tuple_composite_type((List<Bytes32>) bytes);
```

Comment:
look at you using instanceof instead of suppressing the warning.

Reply:
Don't be too impressed. I'm suppressing the warning as well.